doc/en/build.xml

```diff
@@ -96,12 +96,6 @@
   <target name="full" depends="docguide,user,developer">
   </target>
 
-  <target name="sphinx-ignore-warnings" if="sphinx.available">
```

Comment:
Ouch ouch, this introduced a regression... the build is no more incremental, it takes almost a minute now to build the docs. I believe the reason is due to the lack of the arg line element in the "sphinx" target.

Reply:
See if I can work it today.

Reply:
@aaime Can't reproduce. 

A full build from clean (`mvn install` - so the HTML outputs) on my VM takes 2 min 14 sec (all of the `en` docs - docguide, user, developer + the swagger bits). A subsequent build takes about 12 seconds. If I just touch the `user/source/index.rst` file, a build takes about 16 seconds. 

It looks like its doing incremental updates to me. 

Do you have something else?